modules/lunamediahbBidAdapter.js

```diff
@@ -70,7 +70,6 @@ export const spec = {
         schain: bid.schain || {},
       };
       const mediaType = bid.mediaTypes
-      
```

Comment:
Build was complaining about this line, so I removed it:

"/home/circleci/Prebid.js/modules/lunamediahbBidAdapter.js
  73:1  error  Trailing spaces not allowed  no-trailing-spaces"